src/test/scala/com/exasol/cloudetl/kafka/KafkaConsumerPropertiesTest.scala

```diff
@@ -151,7 +165,7 @@ class KafkaConsumerPropertiesTest extends AnyFunSuite with BeforeAndAfterEach wi
     val thrown = intercept[IllegalArgumentException] {
       KafkaConsumerProperties(properties).getProperties()
     }
-    assert(thrown.getMessage() === errorMessage("SCHEMA_REGISTRY_URL"))
+    assert(thrown.getMessage === errorMessage("SCHEMA_REGISTRY_URL"))
```

Comment:
Please revert.

Reply:
We adopted to keep parantheses for method even if they do not accept parameters, this way it is easier to distinguish between variables and methods with no parameters.

Reply:
Ok

Reply:
I reverted all removed parentheses, but found that usage of methods with/without parentheses are currently mixed:

https://github.com/exasol/kafka-connector-extension/blob/0cfa53936c32d286af57d2bbcd45d7725b361a1e/src/test/scala/com/exasol/cloudetl/kafka/KafkaConsumerPropertiesTest.scala#L35
 
https://github.com/exasol/kafka-connector-extension/blob/0cfa53936c32d286af57d2bbcd45d7725b361a1e/src/test/scala/com/exasol/cloudetl/kafka/KafkaConsumerPropertiesTest.scala#L154